src/hssm/distribution_utils/dist.py

```diff
@@ -114,16 +114,22 @@ class SSMRandomVariable(RandomVariable):
         """WFPT random variable."""
 
         name: str = "SSM_RV"
-
-        # NOTE: This is wrong at the moment, but necessary
         # to get around the support checking in PyMC that would result in error
-        ndim_supp: int = 0
+        ndim_supp: int = 1
```

Comment:
can you motivate this change?

The comment above didn't change but this setting did now :). 

Also not sure why this change is related to this pr.

Reply:
This is a change from a previous PR that's already merged. The number of PRs were getting out of hand lol